scripts/renderers.py

```diff
@@ -76,7 +76,8 @@ def filter_and_sort_toggles(self, toggles_info_flattened_dicts, toggle_type_filt
                     data_to_render.append(toggle_dict)
 
         # sort data by either annotation_name or state_name
-        sorting_key = lambda datum: datum.get("name", "")
+        # Without the `or ""` the sorting key can be None if name is set to None explicitly.
+        sorting_key = lambda datum: datum.get("name", "") or ""
```

Comment:
```suggestion
        sorting_key = lambda datum: datum.get("name") or ""
```
Don't even need the default, right?